Tyk single dc (#50)
TykTechnologies/tyk-charts: components/tyk-bootstrap/README.md

## Tyk Bootstrap

## Introduction

> This chart helps with the bootstrapping of the Tyk Dashboard by creating a job which provisions the cluster with all the required secrets/configmaps/HTTP initalization calls [Kubernetes](https://kubernetes.io/) cluster using the [Helm](https://helm.sh/) package manager.

Reviewer: ```suggestion
This chart helps with the bootstrapping of the Tyk Dashboard by creating a job which provisions the cluster with all the required secrets/configmaps/HTTP initialization calls on [Kubernetes](https://kubernetes.io/) cluster using the [Helm](https://helm.sh/) package manager.
```

## Prerequisites
* Kuberentes 1.19+
* Helm 3+

## Installing the Chart

To install the chart from the Helm repository in namespace `tyk` with the release name `tyk-bootstrap`:

    helm repo add tyk-helm https://helm.tyk.io/public/helm/charts/
    helm show values tyk-helm/tyk-bootstrap > values-bootstrap.yaml
    helm install tyk-bootstrap tyk-helm/tyk-bootstrap -n tyk --create-namespace -f values-bootstrap.yaml

> The chart provides little benefits by itself and should only be used in conjunction with other charts that also use the

Reviewer: ```suggestion

```

> The chart provides little benefits by itself and should only be used in conjunction with other charts that also use the
> dashboard component.

Reviewer: ```suggestion

```

## Uninstalling the Chart

    helm uninstall tyk-bootstrap -n tyk

This removes all the Kubernetes components associated with the chart and deletes the release.

## Upgrading Chart

>     helm upgrade tyk-bootstrap tyk-bootstrap -n tyk

Reviewer: ```suggestion
    helm upgrade tyk-bootstrap tyk-helm/tyk-bootstrap -n tyk
```

## Configuration
See [Customizing the Chart Before Installing](https://helm.sh/docs/intro/using_helm/#customizing-the-chart-before-installing). To get all configurable options with detailed comments:

>     helm show values tyk-bootstrap > values-bootstrap.yaml

Reviewer: ```suggestion
    helm show values tyk-helm/tyk-bootstrap > values-bootstrap.yaml
```
